devctl/config/devctl_json_config.json

```diff
@@ -0,0 +1,77 @@
+{
+	"name": "Devctl",
+	"org": "HEMSaw",
+	"site": "OK1",
+	"user": "devctl",
+        "db" : {
+            "type" : "MongoDB",
+            "protocol" : "mongodb",
+            "host" : "mongodb",
+            "port" : 27017,
+            "user": "",
+            "password": ""
+    },
+	"interfaces" : [
+		 {
+			"enable" : "Yes",
+			"device" : 
+				{
+					"device_uid" : "HEMSaw-123456",
```

Comment:
Does this device_uid need to change to match the machine? If so, then we will need to do a sed replace on the file. I can implement this, but I need to know if it is needed to be specific or if it is a placeholder.

Reply:
The device_uid needs to change to match the machine and it needs to be specific for each device.